package.json

```diff
@@ -28,10 +28,13 @@
     "cli": "env NPM_RUN_CLI=1"
   },
   "dependencies": {
-    "ttf2woff2": "^2.0.3",
-    "gulp-util": "^3.0.7",
     "bufferstreams": "^1.1.0",
-    "readable-stream": "^2.0.4"
+    "gulp-util": "^3.0.7",
```

Comment:
Why is gulp-util still in the dependencies?

Reply:
My bad, forgot the most important thing...